pexec/managed_process.go

```diff
@@ -127,6 +127,12 @@ func (p *managedProcess) Start(ctx context.Context) error {
 	default:
 	}
 
+	if p.cwd != "" {
+		if _, err := os.Stat(p.cwd); err != nil {
+			return fmt.Errorf("error with current working directory: %w", err)
```

Comment:
Given the finicky nature of relative paths, I'd expect this error message to have some more context:
* The process [working directory](https://pkg.go.dev/os#Getwd) (or the error string that method can return)
* The `p.cwd` itself

So if there is an error, a human can determine what path we attempted to check for existence.

Reply:
nevermind, it's obviously helpful in the case relative paths

so now it's a matter of naming everything in a way that's not confusing, since we have a "present" cwd and "future" cwd :upside_down_face:

Reply:
https://github.com/viamrobotics/goutils/pull/276/commits/5a5d3907291531729bb7fa653df6438792aef6cf